.idea/.gitignore

```diff
@@ -0,0 +1,3 @@
+# Default ignored files
```

Comment:
You know what this file is used for right?

Reply:
"Yes,
Not think I should use it in my project (correct me if I'm wrong)"

Please reply in the thread itself
*Don't
Why not? It is 20 sec and removes unnecessary files